reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -250,13 +273,14 @@ def fit(self, df):
 
         logger.info("Done training")
 
-    def score(self, test, remove_seen=False):
+    def score(self, test, remove_seen=False, normalize=False):
         """Score all items for test users
 
         Args:
             test (pd.DataFrame): user to test
             remove_seen (bool): flag to remove items seen in training from recommendation
-
+            normalize (bool): flag to normalize scores to be in the same scale as the original ratings
+ 1
         Returns:
             np.ndarray
         """
```

Comment:
I think we can remove the todo line `# TODO: only compute scores for users in test`

Reply:
That is still an improvement that would dramatically reduce computation cost, unfortunately there's a bit of book-keeping necessary to pull it off correctly. Maybe logging it as an issue for future enhancement would be better than inline comments?